ScoutSuite/output/html.py

```diff
@@ -67,66 +81,47 @@ def prepare_html_report_dir(self):
 
 class Scout2Report(HTMLReport):
```

Comment:
Could be a good time to remove the remaining occurrences of `scout2`

Reply:
No problem, will do it!